Spelling: Site wide, matched (#4170)
WeblateOrg/weblate: docs/admin/access.rst

.. image:: /images/project-access.png

To allow access to this project, you have to add the privilege either

> directly to the given user, or group of users in the Weblate admin interface,

Reviewer: Historically, we use Django admin interface. The long-term plan is to replace it for normal management operations by Weblate one, so this is wrong - we still use Django admin interface here.

See https://github.com/WeblateOrg/weblate/issues/2363 for more details.

> directly to the given user, or group of users in the Weblate admin interface,

Author: ```suggestion
directly to the given user, or group of users in the Django admin interface,
```

You can set up Weblate to automatically add users to groups based on their
e-mail addresses. This automatic assignment happens only at the time of account creation.

> This can be set up in the Weblate admin interface for each group (in the

Reviewer: Same as above, IMHO should be reverted.

> This can be set up in the Weblate admin interface for each group (in the

Author: ```suggestion
This can be set up in the Django admin interface for each group (in the
```

.. note::

   Only first rule which is set is being used. So if you set all
   Only the first rule will be used. So if you set all of
   :guilabel:`Component list`, :guilabel:`Components` and :guilabel:`Project`,
   only :guilabel:`Component list` is being applied.
   only :guilabel:`Component list` is the only one applied.

> Additional step is performed if checking permission for the translation:

Author: what does "checking" mean here?

Reviewer: Checking whether user has access to a translation.

An additional step is performed if checking permission for the translation:

> 4. :guilabel:`Languages` are matches against scope of translations, if not set, this matches no

Author: is matched against?
Can "if not set" be avoided?

Reviewer: Group languages are matched against the translation languages. If no group languages are set, it matches none languages (this can be still useful to grant access to a project or component, but not to actual translations.

> 4. :guilabel:`Languages` are matches against scope of translations, if not set, this

Author: ```suggestion
4. :guilabel:`Languages` are matched against the scope of translations if set, if not set, this
```

Managing users and groups
-------------------------

> All users and groups can be managed using the Weblate admin interface,

Reviewer: Same as above, IMHO should be reverted.

> All users and groups can be managed using the Weblate admin interface,

Author: ```suggestion
All users and groups can be managed using the Django admin interface,
```

---------------------

By choosing :guilabel:`Custom` as :guilabel:`Access control`, Weblate will stop

> All users and groups can be managed using the Django admin interface,

Reviewer: "Weblate will stop managing access for a given project" seems removed without any replacement (and the first part of the sentence is now unfinished).

> All users and groups can be managed using the Django admin interface,

Reviewer: ```suggestion
managing access for a given project, and all users and groups can be managed using the Django```

> managing access for a given project, and you can set up custom rules in the Weblate

Reviewer: Same as above, IMHO should be reverted.

> managing access for a given project, and you can set up custom rules in the Weblate

Author: ```suggestion
All users and groups can be managed using the Django admin interface,
```
```suggestion
managing access for a given project, and you can set up custom rules in the Django
```

    Update the internal repository [`Administration`, `Manage repository`]

Site wide privileges
    Use management interface

>     Add language definitions

Reviewer: These should match in app permission names, but it seems to be already out of sync now since https://github.com/WeblateOrg/weblate/pull/2895:

https://github.com/WeblateOrg/weblate/blob/69aa093dfd7c74e94fe2b04e181c31bd54e0bdbb/weblate/auth/data.py#L123-L142

Author: Changed those. I like entire, but in English /project wide/site wide is neat.
